test/src/test/groovy/org/openremote/test/protocol/http/HttpClientProtocolTest.groovy

```diff
@@ -235,7 +235,7 @@ class HttpClientProtocolTest extends Specification implements ManagerContainerTr
                     def yearMonthDay = doDynamicTimeReplace('%TIME:yyyy-MM-dd%', getInstantTimeOf(container))
                     def epochMillis = getInstantTimeOf(container).plus(1, ChronoUnit.HOURS).toEpochMilli()
                     def epochSeconds = getInstantTimeOf(container).minus(50, ChronoUnit.DAYS).getEpochSecond()
-                    def localInstant = LocalDateTime.ofInstant(getInstantTimeOf(container), ZoneId.systemDefault())
+                    def localInstant = getInstantTimeOf(container).atZone(ZoneId.systemDefault()).toLocalDateTime()
```

Comment:
The `localInstant` is no longer an `Instant` with this change so it could use a better name.